types/yup-locale-pl/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for yup-locale-pl 0.1
+// Project: https://github.com/bkostrowiecki/yup-locale-pl
+// Definitions by: Radosław Grochowski <https://github.com/radoslawgrochowski>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LocaleObject } from 'yup';
+declare const locale: LocaleObject;
+export default locale;
```

Comment:
```suggestion
import type { LocaleObject } from 'yup';
declare const locale: LocaleObject;
export = locale;
```
IMO it's CJS module, it will work through compatbity, eg. with 'esModuleInterop' option (which is defaut for ts projects):
https://github.com/bkostrowiecki/yup-locale-pl/blob/main/locale.js#L51

Reply:
Thanks, you're right